docs/reference/troubleshooting/exemptions-and-package-updates.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Exemption and Package Updates (5m)
```

Comment:
nit: none of our other docs have read-time so might suggest we drop for now to align (assuming that was the intent of 5m):
```suggestion
title: Exemption and Package Updates
```

cc @pauldipietro [this](https://docs.astro.build/en/recipes/reading-time/) could be something to look at if we wanted to add read-time more universally?